src/modules/poweraccent/PowerAccent.UI/Selector.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation
+// Copyright (c) Microsoft Corporation
```

Comment:
What is this change. Did you use another encoding?

Reply:
Fixed! File encoding was messed up.
Thank you!